lib/core/time.toit

```diff
@@ -599,7 +601,21 @@ class TimeInfo:
           break
     return "$(year)-$(%02d month)-$(%02d day)T$(%02d h):$(%02d m):$(%02d s)$fraction$(is-utc ? "Z" : "")"
 
-  /** See $super. */
+  /**
+  Converts this instance to a date string of the form 'yyyy-mm-dd'.
+  Hours, minutes and seconds are ignored.
+  */
+  to-date-string -> string:
+    return "$year-$(%02d month)-$(%02d day)"
+
+  /**
+  Converts this instance to an string conforming to RFC 3339, which is a subset
```

Comment:
I think `stringify` should not make any guarantees.
Users should use `to-iso8601-string` (or `to-rfc-3339-string` if that exists).